index.html

```diff
@@ -48,7 +57,180 @@ <h1>I’m Nurbol<br>Glad to see<br> you!</h1>
+                        </div>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="bottom">
+                        <ul class="tags">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li></ul>
+                        <button class="action">See Project</button>
+                        </div>
+                    </div>
+            </div>
+
+<!--2 work card-->   
+
+            <div class="works-card">
+                <img src="worksimg\Snapshoot Portfolio@2x (1).png">
+                <div class="info-block">
+                    <div class="top">
+                        <h2 id="eight">Multi-Post<br>Stories</h2>
+                        <ul class="frame">
+                            <li id="client">CANOPY</li>
+                            <li id="dot"> </li>
+                            <li id="role">Back End Dev</li>
+                            <li id="dot"> </li>
+                            <li id="year">2015</li></ul>
+                    </div>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="bottom">
+                    <ul class="tags">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li></ul>
+                    <button class="action">See Project</button>
+                </div>
+            </div>
+        </div>
+
+<!--3 work card-->
+
+<div class="works-card">
+    <img src="worksimg/Snapshoot Portfolio@2x (2).png">
+    <div class="info-block">
+        <div class="top">
+            <h2 id="six">Tonic</h2>
+            <ul class="frame">
+                <li id="client">CANOPY</li>
+                <li id="dot"> </li>
+                <li id="role">Back End Dev</li>
+                <li id="dot"> </li>
+                <li id="year">2015</li></ul>
+        </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="bottom">
+            <ul class="tags">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li></ul>
+            <button class="action">See Project</button>
+        </div>
+    </div>
+</div>
+
+<!--4 work card-->
+
+    <div class="works-card">
+        <img src="worksimg/Snapshoot Portfolio@2x (3).png">
+        <div class="info-block">
+            <div class="top">
+            <h2 id="eight">Multi-Post<br>Stories</h2>
+            <ul class="frame">
+                <li id="client">CANOPY</li>
+                <li id="dot"> </li>
+                <li id="role">Back End Dev</li>
+                <li id="dot"> </li>
+                <li id="year">2015</li></ul>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```

Comment:
- [ ] This is looking great now, but it can be better. I think the `font-weight` on this paragraph and every other instance of it is quite much 👇 

![image](https://user-images.githubusercontent.com/99715640/212458903-671e91f4-25d2-408d-a5a9-77579328806f.png)

Reply:
i did use  same properties as in the template [https://drive.google.com/file/d/1DmqTciX1-9-vVz0273-9iHM_2GjfLuFx/view?usp=share_link](url)